### PR TITLE
[Merged by Bors] - feat: port Combinatorics.SetFamily.LYM

### PR DESCRIPTION
**Pull request by mo271**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by mo271** [comment]
Should the filename be `Mathlib/Combinatorics/SetFamily/LYM.lean` instead of `Mathlib/Combinatorics/SetFamily/Lym.lean`?

File: Mathlib/Combinatorics/SetFamily/Lym.lean [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+variable {𝕜 α : Type _} [LinearOrderedField 𝕜]
+
+namespace Finset
+
+/-! ### Local LYM inequality -/
+
+
+section LocalLym
+ -- porting note: added `Decidable` for ⊆
```
Comment (mo271) [inline Mathlib/Combinatorics/SetFamily/Lym.lean:63]: not sure how to handle the fact that I couldn't do `card_mul_le_card_shadow_mul` without the Decidable assumption for `⊆`. Is there some way around it?
Should this be placed elsewhere?

**Comment by ChrisHughes24** [comment]
> Should the filename be `Mathlib/Combinatorics/SetFamily/LYM.lean` instead of `Mathlib/Combinatorics/SetFamily/Lym.lean`?

I think so

**Comment by ChrisHughes24** [comment]
bors merge

**Pull request closed without merging**